configure

```diff
@@ -1436,6 +1447,7 @@ Fine tuning of the installation directories:
   --sysconfdir=DIR        read-only single-machine data [PREFIX/etc]
   --sharedstatedir=DIR    modifiable architecture-independent data [PREFIX/com]
   --localstatedir=DIR     modifiable single-machine data [PREFIX/var]
+  --runstatedir=DIR       modifiable per-process data [LOCALSTATEDIR/run]
```

Comment:
Ditto here.  The only diff in the `configure` that's relevant is the last one beginning here: https://github.com/python/cpython/pull/12825/files#diff-e2d5a00791bce9a01f99bc6fd613a39dR7369

Reply:
same as above: these are autotools-generated, then just not need to be reviewed.

Reply:
I know: My point is you can drop this changes from the PR because they're not relevant even as autogenerated autotools stuff.  Better to just keep the relevant changes even those files.

Reply:
using newer autotools fixed this by itself.